static/js/components/search/CustomNoHits.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from 'react';
+import { NoHits } from 'searchkit';
+
+export default class CustomNoHits extends NoHits {
+  render() {
+    if ((this.hasHits() || this.isInitialLoading() || this.isLoading()) && !this.getError()) {
+      return null;
+    }
+
+    if (this.getError()) {
+      const suggestion = this.getSuggestion();
+      const query = this.getQuery().getQueryString();
+      let infoKey = suggestion ? "NoHits.NoResultsFoundDidYouMean" : "NoHits.NoResultsFound";
+
+      return (
+        <div className="no-hits">
+          {this.translate(infoKey, {query:query, suggestion:suggestion})}
```

Comment:
If the query is empty, the string ends up as "No results found for .", which is exactly what I saw when I tested it:
![screen shot 2016-10-26 at 3 30 49 pm](https://cloud.githubusercontent.com/assets/132355/19742096/8259c486-9b91-11e6-9788-cb1ee98a885f.png)

Maybe treat an empty query as a special case, and ask the user to search for something? @roberthouse54, do you have any thoughts on how this should be handled?